Tools/CLSGenerator.sh

```diff
@@ -0,0 +1,180 @@
+                # Replicating the broad copy from original batch
+                copy_dir_recursive "../Addons/Custom Lua Support/." "./Client/"; copy_status=$?
+                choice_successful=true
+                ;;
+            2) # Headgears
+                item_failed=false
+                copy_file "$sourceDataPath/cls/accessoryid.lub" "$destinationDataPath/cls/accessoryid.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/accname.lub" "$destinationDataPath/cls/accname.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/accname_f.lub" "$destinationDataPath/cls/accname_f.lub" || item_failed=true
+                copy_file "$sourceDataPath/datainfo/TB_Layer_Priority.lub" "$destinationDataPath/datainfo/TB_Layer_Priority.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            3) # Random Options
+                item_failed=false
+                copy_file "$sourceDataPath/cls/enumvar.lub" "$destinationDataPath/cls/enumvar.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/randomoption.lub" "$destinationDataPath/cls/randomoption.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/addrandomoption_f.lub" "$destinationDataPath/cls/addrandomoption_f.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            4) # HatEffects
+                item_failed=false
+                copy_file "$sourceDataPath/hateffectinfo/hateffectinfo.lub" "$destinationDataPath/hateffectinfo/hateffectinfo.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/hateffectinfo.lub" "$destinationDataPath/cls/hateffectinfo.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            5) # NPC/Mob and Pets
+                item_failed=false
+                copy_file "$sourceDataPath/cls/jobname.lub" "$destinationDataPath/cls/jobname.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/jobname_f.lub" "$destinationDataPath/cls/jobname_f.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/npcidentity.lub" "$destinationDataPath/cls/npcidentity.lub" || item_failed=true
+                copy_file "$sourceDataPath/cls/petinfo.lub" "$destinationDataPath/cls/petinfo.lub" || item_failed=true
+                copy_status=$([[ "$item_failed" = true ]] && echo 1 || echo 0)
+                choice_successful=true
+                ;;
+            6) # Lapine Boxes
+                item_failed=false
+                copy_file "$sourceDataPath/datainfo/lapineddukddakbox.lub" "$destinationDataPath/datainfo/lapineddukddakbox.lub" || item_failed=true
```

Comment:
Remove lines 96 and 97

Reply:
Done on https://github.com/llchrisll/ROenglishRE/pull/100/commits/0e67ea0ed5152e0ed3775913935022d1643dbb67#diff-51a4c473a095a632f039964dc2a80c0e4c5794873ab1cfccfcbdffdf7114bb3bL96-L97